Filtered_kernel/include/CGAL/Lazy_kernel.h

```diff
@@ -480,6 +480,18 @@ class Lazy_kernel_base
 
   };
 
+  struct Less_xyz_3 : public BaseClass::Less_xyz_3
+  {
+    typedef typename Kernel_::Point_3 Point_3;
+
+    using BaseClass::Less_xyz_3::operator();
```

Comment:
What other calls are you trying to catch with this?

Reply:
Indeed it is useless since I expended it in the operator().